src/utils.js

```diff
@@ -612,3 +627,21 @@ export const getDecodedParams = (params) =>
     }
     return acc
   }, {})
+
+export const getClassNames = (styles, classes) => {
+  // Generates relevant class, with classes using the provided styles
+  const styledClasses = []
+  classes.forEach((className) => {
+    if (UNSTYLED_CLASS_NAMES.includes(className)) {
+      // Styles related to colours - only use the original name so that our colours take precedence
+      styledClasses.push(className)
+    } else if (OVERRIDDEN_CLASS_NAMES.includes(className)) {
+      // Styles which affect our inserted styles - require both our css and the imported css
+      styledClasses.push(className)
+      styledClasses.push(styles[className])
+    } else {
+      styledClasses.push(styles[className])
+    }
+  })
+  return classNames(styledClasses)
+}
```

Comment:
wdyt about rewriting it so that the intention is clearer?

```ts
  return classes
    .filter((className) => !UNSTYLED_CLASS_NAMES.includes(className))
    .map((className) => styles[className])
    .concat(_.intersection(classes, OVERRIDDEN_CLASS_NAMES))
```

Reply:
Makes sense, https://github.com/isomerpages/isomercms-frontend/pull/895/commits/6184d6c519cb1d039d70f6606228740a773b0bd5